Assets/PurrNet/CHANGELOG.md

```
@@ -1,3 +1,553 @@
+# [2.0.0-beta.1](https://github.com/feiyuren233/PurrNet/compare/v1.12.4...v2.0.0-beta.1) (2026-04-08)
+
+
+### Bug Fixes
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)